components/sandbox/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../base
```

Comment:
this is wrong path, kustomize build is failing. I suggest instead of adding this kustomize file, that we rename base folder to kubesaw-common and change `argo-cd-apps/base/all-clusters/infra-deployments/kubesaw-common/kubesaw-common.yaml` application to point to `components/sandbox/kubesaw-common` folder.

Having a folder named the same way as the ArgoCD application is kind of a pattern we use, making it simpler to identify the folder associated with the application

Reply:
OK, I just directly pointed to base in worked, going to rename base

Reply:
I like that !

Reply:
done